pkg/gofr/context_test.go

```diff
@@ -36,3 +38,54 @@ func Test_newContextSuccess(t *testing.T) {
 	assert.Equal(t, map[string]string{"key": "value"}, body, "TEST Failed \n unable to read body")
 	require.NoError(t, err, "TEST Failed \n unable to read body")
 }
+
+func TestContext_Trace(t *testing.T) {
+	ctx := Context{
+		Context: context.Background(),
+	}
+
+	span := ctx.Trace("Some Work")
+	defer span.End()
+
+	assert.NotNil(t, span)
```

Comment:
```suggestion
	require.NotNil(t, span)

	defer span.End()
```
Otherwise you will get a panic

But is that even possible? Trace code always return a span

Said otherwise, what are you trying to test?

Reply:
I guess checking whether context is modified or not ... because of addition of tracer would be a better option.